firefox-ios/firefox-ios-tests/Tests/ClientTests/Mocks/MockDataCleaner.swift

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+@testable import Client
+
+class MockCookiesClearable: CookiesClearable {
+    var isSucceed: Success?
+
+    override func clear() -> Success {
+        isSucceed = succeed()
```

Comment:
nit: I prefer the simple call count checks instead of adding this logic here, since we're essentially testing that the call is triggered. However, this isn't something we standardized on the team, so not a blocker.